includes/class-sensei-groups-landing-page.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * File Sensei_Groups_Landing_Page class.
+ *
+ * @package sensei-lms
+ * @since x.x.x
```

Comment:
JFYI, we are now using `$$next-version$$` instead of `x.x.x`. It's ok to use it for the next release, but we will stop using `x.x.x` after that.

p6rkRX-3BS-p2

Reply:
Thanks, missed this !!